source/kraken/readme.md

```diff
@@ -0,0 +1,161 @@
+# Kraken
+Kraken is the public transport trip planner and much more, it can provide schedules, timetables and expose public
+transport data via
+[PTReferential](https://github.com/CanalTP/navitia/blob/dev/documentation/rfc/ptref_grammar.md).
+It also provide:
+  - (bad) street network routing that will be replaced by [asgard](https://github.com/CanalTP/asgard)
+  - autocomplete that will be replaced by [mimir](https://github.com/CanalTP/mimirsbrunn)
+
+
+## Configuration
+Kraken can be configured by multiple way:
+  - configuration file
+  - command line arguments
+  - environment variables
+
+### Configuration file
+Kraken will try to load a configuration file named the same way as the binary with a `.ini` prefix in the
+current directory, so by default it will search for a `kraken.ini` file. Alternatively the file can be passed as
```

Comment:
```suggestion
current directory, so by default it will search for a `kraken.ini` file. Alternatively the file can be passed in as
```

Reply:
`in as`? `as` should be removed no?

Reply:
as in the idiomatic expression 'pass in' - https://idioms.thefreedictionary.com/pass+in. But I'm being nitpicking here.